src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -118,15 +118,21 @@ extern int halide_hexagon_set_performance_mode(void *user_context, halide_hexago
 extern int halide_hexagon_set_performance(void *user_context, halide_hexagon_power_t *perf);
 // @}
 
-/** Set the default priority for Halide Hexagon user threads:
+/** Set the default priority/stack_size for Halide Hexagon user threads:
  *   - Valid priority values range from 1 to 255
  *   - Smaller number for higher priority
  *   - The highest priority for a user thread is 1
  *   - Priority 0 is reserved for OS usage
- * If this routine is not called, the priority will default to 100.
- * This is intended to be called before dispatching any pipeline. */
+ * If halide_hexagon_set_thread_priority routine is not called, the priority
```

Comment:
@aankit-ca  - With PipelineContext removed ``halide_remote.cpp``, I don't think the plumbing for ``halide_hexagon_set_thread_priority`` to get the priority set for each thread in the parallel thread pool exists any more. We should either completely remove this function or change it to use ``remote_session_control`` where available and fall back to its current approach now.(I don't know if many people use it though). We should also use the new thread_params mechanism to set the thread priority and/ or stack size.

```
// to set only the priority
halide_hexagon_set_remote_thread_params(NULL, 10, -1);
// to set only the stack size
halide_hexagon_set_remote_thread_params(NULL, -1, 1024 * 32);
// to set both
halide_hexagon_set_remote_thread_params(NULL, 10, 1024 * 32);
```